src/main/java/org/example/exceptions/Entity.java

```diff
@@ -0,0 +1,16 @@
+package org.example.exceptions;
+
+public enum Entity {
+
+    JOBROLERESPONSE("JobRoleResponse");
```

Comment:
Is there a reason this is here?

Reply:
Used in the JobRoleService for if JobRoleResponses is Empty